### PR TITLE
use django's translation for pluralizations in invoices

### PR DESCRIPTION
**Pull request by nickpell**
... instead of trying to handle pluralization ourselves

### CODE REVIEW

**Comment by nickpell** [comment]
@esoergel

File: corehq/apps/accounting/invoicing.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 from django.db import transaction
 from django.db.models import F, Q, Min, Max
-from django.utils.translation import ugettext as _
+from django.utils.translation import ugettext as _, ungettext
```
Comment (esoergel) [inline corehq/apps/accounting/invoicing.py]: I had to look this one up.  Fancy, looks nice!

File: corehq/apps/accounting/invoicing.py [hunk 2]
```diff
@@ -418,9 +418,12 @@ def base_description(self):
     @property
     def unit_description(self):
         if self.is_prorated:
-            return _("%(num_days)s day%(pluralize)s of %(plan_name)s Software Plan.") % {
+            return ungettext(
+                "%(num_days)s day of %(plan_name)s Software Plan.",
+                "%(num_days)s days of %(plan_name)s Software Plan.",
+                self.num_prorated_days
+            ) % {
```
Comment (esoergel) [inline corehq/apps/accounting/invoicing.py]: As an aside, do you have a strong preference for `"Hello %(name)s" % name` over `"Hello {name}".format(name)`?  I tend to prefer the latter, as I find it more readable (translators can get confused by that trailing `s`) and flexible, but curious if you have arguments against it.

**Comment by esoergel** [comment]
Do you know if transifex has any separate handling for instances like this?  Seems like it'd be a shame if it dramatically increased the amount of work our translators need to do.

**Comment by esoergel** [comment]
I just dug in to it a bit, and I see that they're stored with a `msgid_plural` attr in the `.po` files, but I wasn't able to find any of the pluralized strings on transifex, and searching for `pluralized: yes` shows no results.  Looks like something might be wrong with our config.

**Pull request merged**